ui/page/settings/app_settings_page.go

```diff
@@ -647,6 +680,83 @@ func (pg *AppSettingsPage) HandleUserInteractions() {
 			pg.ParentWindow().ShowModal(currentPasswordModal)
 		}
 	}
+
+	if pg.backupDEX.Clicked() {
+		// Show modal asking for dex password and then reveal the seeds.
+		dexPasswordModal := modal.NewCreatePasswordModal(pg.Load).
+			EnableName(false).
+			EnableConfirmPassword(false).
+			Title(values.String(values.StrDexPassword)).
+			SetPositiveButtonCallback(func(_, password string, pm *modal.CreatePasswordModal) bool {
+				var err error
+				defer func() {
+					if err != nil {
+						pm.SetError(err.Error())
+						pm.SetLoading(false)
+					}
+				}()
+
+				dexc := pg.AssetsManager.DexClient()
+				err = dexc.Login([]byte(password))
+				if err != nil {
+					return false
+				}
+
+				seed, err := dexc.ExportSeed([]byte(password))
+				if err != nil {
+					return false
+				}
+
+				pg.showDEXSeedModal(seed)
+				return err == nil
+			})
+
+		dexPasswordModal.SetPasswordTitleVisibility(false)
+		pg.ParentWindow().ShowModal(dexPasswordModal)
+	}
+}
+
+func (pg *AppSettingsPage) handleCopyEvent(gtx C) {
```

Comment:
I don't think this is called more than once, doesn't need to be a method. If anything, we may want a util method for copying text to the clipboard.

Reply:
Not called more than once but it's cleaner this way.